OWML.ModLoader/ModSorter.cs

```diff
@@ -0,0 +1,90 @@
+﻿using OWML.Common;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace OWML.ModLoader
+{
+    class ModSorter
+    {
+        public IList<ModDep> SortMods(IList<IModData> mods)
+        {
+            var modList = new List<ModDep>();
+            foreach (var mod in mods)
+            {
+                if (mod.Manifest.Dependencies == null)
+                {
+                    mod.Manifest.SetDependencies(new[] { "None" });
+                }
+
+                if (mod.Manifest.Dependencies.First() != "None")
+                {
+                    modList.Add(new ModDep(mod.Manifest.Name, mod, mod.Manifest.Dependencies));
+                }
+                else
+                {
+                    modList.Add(new ModDep(mod.Manifest.Name, mod));
+                }
```

Comment:
🤔 if Dependencies didn't default to { "none" }, couldn't you collapse these into just `modList.Add(new ModDep(mod.Manifest.Name, mod));` ?

Reply:
ModDep needs the dependencies to sort the mods, but if it's "None" then it doesn't add any dependencies